source/include/marlin/PatchCollections.h

```diff
@@ -0,0 +1,92 @@
+#ifndef PatchCollections_h
+#define PatchCollections_h 1
+
+#include "marlin/Processor.h"
+#include "marlin/EventModifier.h"
+#include "lcio.h"
+#include <string>
+#include <vector>
+#include <map>  // pair
```

Comment:
```suggestion
#include <utility>  // pair
#include <memory>
```
`utitlity` is probably the smaller header than `map`, and `pair` is defined there.